### PR TITLE
Add curriculum page

### PR DESCRIPTION
**Pull request by ruioliveira02**
Please give feedback on the layout of the page and the text content.

I based it off an internal document, but I am unsure of some requirements.

### CODE REVIEW

File: pages/curriculum.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    {belts.map((b) => (
+      <Belt key={b.colour} {...b} />
+    ))}
+
+    <Footer bgColor="dark" fgColor="white" />
+  </>
+);
+
+export async function getStaticProps() {
```
Comment (nelsonmestevao) [inline pages/curriculum.js:83]: Why don't you just import he JSON file?

Comment (ruioliveira02) [reply pages/curriculum.js:83]: So that the data fetching occurs during the build stage and not on the client side

File: data/curriculum.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    "description": "Neste primeiro contacto com o Ninja, procuramos ensinar conceitos básicos de programação, sem necessidade de dominar qualquer conceito.",
+    "image": "white.png",
+    "requirements": [
+      "Ter iniciado os conceitos básicos de programação",
+      "Ter começado pelo menos um projeto em Scratch"
+    ]
+  },
+  {
+    "colour": "Amarelo",
+    "description": "Após o primeiro cinturão, procura-se aprimorar os primeiros conceitos que foram introduzidos ao Ninja nas sessões anteriores, de forma a que ele se sinta mais confortável em usar o Scratch e começe já a criar projetos um pouco mais elaborados. ",
```
Comment (RuiL1904) [inline data/curriculum.json:13]:
```suggestion
    "description": "Após o primeiro cinturão, procura-se aprimorar os primeiros conceitos que foram introduzidos ao Ninja nas sessões anteriores, de forma a que ele se sinta mais confortável em usar o Scratch e começe já a criar projetos um pouco mais elaborados.",
```

File: data/curriculum.json [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      "Ter concluído pelo menos um projeto em Scratch"
+    ]
+  },
+  {
+    "colour": "Azul",
+    "description": "Para obter o cinturão azul, o Ninja já deve possuir uma maior autonomia com o Scratch, assim como ter concluído outros 2 projetos.",
+    "image": "blue.png",
+    "requirements": [
+      "Manipular as variáveis da forma correta",
+      "Apresentar um projeto a um mentor",
```
Comment (RuiL1904) [inline data/curriculum.json:27]:
```suggestion
      "Apresentar um projeto a um Mentor",
```

File: data/curriculum.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    "image": "yellow.png",
+    "requirements": [
+      "Maior autonomia com os conceitos básicos",
+      "Saber aplicar condicionais e ciclos",
+      "Ter concluído pelo menos um projeto em Scratch"
+    ]
+  },
+  {
+    "colour": "Azul",
+    "description": "Para obter o cinturão azul, o Ninja já deve possuir uma maior autonomia com o Scratch, assim como ter concluído outros 2 projetos.",
```
Comment (RuiL1904) [inline data/curriculum.json:23]:
```suggestion
    "description": "Para obter o cinturão Azul, o Ninja já deve possuir uma maior autonomia com o Scratch, assim como ter concluído outros 2 projetos.",
```

File: data/curriculum.json [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    "image": "blue.png",
+    "requirements": [
+      "Manipular as variáveis da forma correta",
+      "Apresentar um projeto a um mentor",
+      "Terminar outros 2 projetos em Scratch"
+    ]
+  },
+  {
+    "colour": "Verde",
+    "description": "O cinturão verde poderá ser obtido depois de um período de transição entre a plataforma Scratch e uma linguagem de programação mais textual, que ficará a critério do Mentor e do Ninja. Recomendam-se linguagens de fácil aprendizagem e compreensão como Python, por exemplo.",
```
Comment (RuiL1904) [inline data/curriculum.json:33]:
```suggestion
    "description": "O cinturão Verde poderá ser obtido depois de um período de transição entre a plataforma Scratch e uma linguagem de programação mais textual, que ficará a critério do Mentor e do Ninja. Recomendam-se linguagens de fácil aprendizagem e compreensão como Python, por exemplo.",
```

File: data/curriculum.json [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    "requirements": [
+      "Aplicar o que aprendeu no Scratch noutra linguagem",
+      "Compreender os conceitos e as aplicações de Listas e Strings",
+      "Terminar 2 projetos na nova linguagem (1 deles pode ser uma adaptação de algum já feito em Scratch, se o Ninja quiser)",
+      "Apresentar um projeto a 2 mentores"
+    ]
+  },
+  {
+    "colour": "Laranja",
+    "description": "Com o cinturão laranja, o ninja aprenderá a utilizar ferramentas da sua linguagem escolhida para criar interfaces ou até jogos simples. O Ninja deverá aprender também sobre os conceitos de I/O, com recurso às bibliotecas do sistema operativo.",
```
Comment (RuiL1904) [inline data/curriculum.json:44]:
```suggestion
    "description": "Com o cinturão Laranja, o Ninja aprenderá a utilizar ferramentas da sua linguagem escolhida para criar interfaces ou até jogos simples. O Ninja deverá aprender também sobre os conceitos de I/O, com recurso às bibliotecas do sistema operativo.",
```

File: data/curriculum.json [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    "requirements": [
+      "Aprender a criar interfaces",
+      "Utilizar módulos ou bibliotecas extra como Pygame, GUIZero, etc.",
+      "Terminar 1 projeto usando interface e 1 adaptação de um projeto feito em Scratch",
+      "Apresentar um projeto a 5 ninjas"
+    ]
+  },
+  {
+    "colour": "Vermelho",
+    "description": "Para este cinturão, o Ninja aprenderá mais sobre Web e deverá criar o seu próprio Website. Para a criação do website, recomenda-se ensinar uma nova linguagem de programação como JavaScript, por exemplo.  Recomenda-se também que o Ninja possua um repositório de código para guardar os seus projetos como GitLab ou GitHub, por exemplo.",
```
Comment (RuiL1904) [inline data/curriculum.json:55]:
```suggestion
    "description": "Para este cinturão, o Ninja aprenderá mais sobre Web e deverá criar o seu próprio Website. Para a criação do Website, recomenda-se ensinar uma nova linguagem de programação como JavaScript, por exemplo.  Recomenda-se também que o Ninja possua um repositório de código para guardar os seus projetos como GitLab ou GitHub, por exemplo.",
```

File: data/curriculum.json [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+    "image": "red.png",
+    "requirements": [
+      "Aprender a usar uma nova linguagem que lhe permita criar o website",
+      "Criar um website",
+      "Apresentar um projeto para uma turma",
+      "Uma biblioteca para fazer web servers (Flask para Python, Sinatra para Ruby, Spark para Java)"
```
Comment (RuiL1904) [inline data/curriculum.json:61]:
```suggestion
      "Aprender a usar uma nova linguagem que lhe permita criar o Website",
      "Criar um Website",
      "Apresentar um projeto para uma turma",
      "Uma biblioteca para fazer Web servers (Flask para Python, Sinatra para Ruby, Spark para Java)"
```

File: data/curriculum.json [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+    ]
+  },
+  {
+    "colour": "Roxo",
+    "description": "Para o Cinturão Roxo, o Ninja deverá aprender sobre o conceito de Regex e também sobre Bases de Dados, visando produzir projetos mais avançados nas linguagens que já aprendeu. Deverá ser também introduzido os conceitos de APIs como uma extensão de módulos de programas",
```
Comment (RuiL1904) [inline data/curriculum.json:66]:
```suggestion
    "description": "Para o cinturão Roxo, o Ninja deverá aprender sobre o conceito de Regex e também sobre Bases de Dados, visando produzir projetos mais avançados nas linguagens que já aprendeu. Deverá ser também introduzido os conceitos de APIs como uma extensão de módulos de programas",
```

**Review by RuiL1904 — APPROVED** [review]

**Pull request merged**